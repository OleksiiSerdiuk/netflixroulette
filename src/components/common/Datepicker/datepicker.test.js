import { render, screen } from '@testing-library/react';
import { Formik } from 'formik';
import DatePickerField from './Datepicker';
import '@testing-library/jest-dom/extend-expect';

describe('datepicker', () => {
  test('renders datepicker snapshot', () => {
    const { asFragment } = render(
      <Formik>
        <DatePickerField
          control='datepicker'
          label='Release Date'
          name='release_date'
        />
      </Formik>
    );
    expect(asFragment(<DatePickerField />)).toMatchSnapshot();
    expect(screen.getByText(/Release Date/i)).toBeInTheDocument();
  });
});
