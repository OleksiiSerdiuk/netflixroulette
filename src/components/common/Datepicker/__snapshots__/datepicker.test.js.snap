// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`datepicker renders datepicker snapshot 1`] = `
<DocumentFragment>
  <div
    class="base-datepicker"
  >
    <span
      class="base-datepicker__label"
    >
      Release Date
    </span>
    <div
      class="react-datepicker-wrapper"
    >
      <div
        class="react-datepicker__input-container"
      >
        <input
          class="base-datepicker__input"
          name="release_date"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
